.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
-extend-ignore = E203, E402, E501, E731, E741, B950
+extend-ignore = E203, E402, E501, E731, E741, B950, W503
```

Comment:
Has a conflict with Black: https://stackoverflow.com/questions/68161741/python-black-formatter-conflict-with-rule-flake8-w503-in-vscode

Reply:
For the record, black is right here: line breaks should happen before operators not after them. Apparently Donald Knuth advocated this position which is why people are switching to it. But it's also the correct way.

Reply:
Right, it seems like they are going to change the best practice https://www.flake8rules.com/rules/W503.html. I didn't know that, thanks!